client/js/components/forms/ProfileForm.js

```diff
@@ -21,37 +22,63 @@ class ProfileForm extends Component {
     const { errors } = this.props;
 
     return (
-      <div>
+      <div id="ProfileForm">
```

Comment:
Can you use class instead of id? I can imagine a situation when someone could use multiple ProfileForm component on a single page. We probably won't do it, but using ID here limits reusability.